rudder-agent/SOURCES/Makefile

```diff
@@ -442,6 +442,10 @@ ifeq ($(INSTALL),ginstall)
 endif
 	cd fusioninventory-agent && PERL_MM_USE_DEFAULT=1 $(INSTALL_DIR)/bin/perl Makefile.PL --default PREFIX=$(RUDDER_DIR)
 	cd fusioninventory-agent && $(MAKE_COMMAND) install DESTDIR=$(DESTDIR)
+	# bug in fusion installer that doesn't set the right owner to some files
+	chown root $(DESTDIR) opt/rudder/share/fusioninventory
```

Comment:
Do you change owner of $DESTDIR three times ? or have you missed a slash  between DESTDIR and the path ?